contrib/drivers/ceph/ceph.go

```diff
@@ -555,17 +555,16 @@ func (d *Driver) getPoolsAttr() (map[string][]string, error) {
 	return poolDetail, nil
 }
 
-func (d *Driver) buildPoolParam(line []string, proper PoolProperties) *map[string]interface{} {
-	param := BuildDefaultPoolParam(proper)
-
-	param["redundancyType"] = line[poolType]
-	if param["redundancyType"] == "replicated" {
-		param["replicateSize"] = line[poolTypeSize]
+func (d *Driver) buildPoolExtras(line []string, extras model.StoragePoolExtraSpec) model.StoragePoolExtraSpec {
+	extras.Advanced["redundancyType"] = line[poolType]
+	if extras.Advanced["redundancyType"] == "replicated" {
```

Comment:
Is "replicated" a Ceph specific redundancy type?  If there are terms like this used in multiple places, it maybe better to use constants to avoid typo.

Reply:
So in ceph every pool can be assigned with two redundancy types: `replicated` and `raid`, and I will add two constants to define them.